src/components/NavPanel/index.js

```diff
@@ -1,40 +1,27 @@
-import style from './style.css'
+import SvgButton from '../SvgButton';
 
-const SvgButton = ({path}) => {
-    
-    function formatPath(d) {
-        return <path strokeLinecap="round" strokeLinejoin="round" strokeWidth={2} d={d} />;
-    };
+const BTN_CLASSES = "text-gray-400 hover:text-gray-100"
 
-    return (
-        <a class="btn" href="#">
-            <svg class="w-8 h-8" xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke="currentColor">
-                { Array.isArray(path)
-                    ? path.map(d => formatPath(d)) 
-                    : formatPath(path)
-                    }
-            </svg>
-        </a>
-    );
-};
-
-const NavPanel = () => (
+const NavPanel = ({store}) => (
     <>
-        <style>{style}</style>
-        <div class="bg-gray-300 w-16 flex flex-col min-h-full relative">
+        <div class="bg-gray-800 flex flex-col min-h-full relative">
```

Comment:
can we use this classes in ul and remove unnecessary div?

Reply:
Let me check

Reply:
Nope. Because of sticky btm btn